frontend/src/pages/articles/my-articles.tsx

```diff
@@ -1,12 +1,44 @@
+import {
+  useAppDispatch,
+  useAppSelector,
+  useEffect,
+} from '~/libs/hooks/hooks.js';
+import { type UserDetailsResponseDto } from '~/packages/users/users.js';
+import { actions as articlesActions } from '~/slices/articles/articles.js';
+
 import { ArticleCard } from './components/components.js';
-import { MOCKED_ARTICLES, MOCKED_USER } from './libs/constants.js';
-
-const MyArticles: React.FC = () => (
-  <>
-    {MOCKED_ARTICLES.map((article) => (
-      <ArticleCard key={article.id} article={article} user={MOCKED_USER} />
-    ))}
-  </>
-);
+import { MOCKED_REACTIONS, MOCKED_TAGS } from './libs/constants.js';
+
+const MyArticles: React.FC = () => {
+  const dispatch = useAppDispatch();
+  const { articles, user } = useAppSelector(({ articles, auth }) => ({
+    articles: articles.articles,
+    user: auth.user,
+  }));
+
+  const author = {
+    firstName: user?.firstName,
+    lastName: user?.lastName,
+  } as UserDetailsResponseDto;
+
+  useEffect(() => {
+    void dispatch(articlesActions.fetchOwn());
+  }, [dispatch]);
+
+  return (
+    <>
+      {articles &&
```

Comment:
`articles &&` can be replaced with optional chaining when mapping over articles. Then we also won’t need a fragment

```js
  articles?.map()
```